test/controllers/questions_controller_test.rb

```diff
@@ -1,22 +1,30 @@
+# frozen_string_literal: true
+
 require './test/test_helper'
 
 class QuestionsControllerTest < ActionDispatch::IntegrationTest
-
   class AuthenticatedTest < ActionDispatch::IntegrationTest
+    def expect_successful_response(expected_response)
```

Comment:
I like creating helper methods, in fact I'd probably create another method for "Actual" "expected" as well in the other tests but this one was more domain specific - I would also consider creating a new class called Controller test with all these helpers, though I don't do minitest so didn't know how to immediately.

Reply:
I also like doing that, but I chose not to because we only have one controller, I usually attempt to abstract things in second or third time the concept arises.

Reply:
We still had a few tests with this same expect template - for me it was a few times, but it's not so much DRY it's just more clear to read - the text actually clutters the tests. It doesn't help me understand the test output

Reply:
I've just pushed a commit to do that! LGTM 🤩